packages/xo-server/src/api/vm.mjs

```diff
@@ -540,6 +540,20 @@ insertCd.resolve = {
 
 // -------------------------------------------------------------------
 
+export function getSecurebootReadiness({ vm }) {
+  return this.getXapi(vm).call('VM.get_secureboot_readiness', vm._xapiRef)
```

Comment:
@Pizzosaure @MathieuRA We need to cache this call (discuss with @stormi about the delay).

Also, we probably need to add a `_forceRefresh` param to bypass the cache.

Reply:
We defined a 30s delay before it's queried again, outside forced refreshes, so I think this would translate into a 30s long cache?

Reply:
Still not fixed, I want a cache on xo-server's side.